src/common/hooks/useGetPlays.js

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect, useContext } from "react";
+import { SearchContext } from "common/search/search-context";
+import { FetchPlaysFilter } from "common/services/request/query/fetch-plays-filter";
+import { FetchPlaysSimple } from "common/services/request/query/fetch-plays";
+
+import { submit } from "common/services/request";
+
+const useGetPlays = () => {
+  const [loading, setLoading] = useState(true);
+  const [plays, setPlays] = useState([]);
+  const [error, setError] = useState(null);
+
+  const { filterPlaysBySearchString, filterPlaysByMultiTagsLevelLang } =
+    FetchPlaysFilter;
+
+  const { searchTerm, filterQuery } = useContext(SearchContext);
+
+  const hasSearchTerm = searchTerm.length > 0;
+  const hasFilterQuery =
+    filterQuery.level.length > 0 ||
+    filterQuery.tags.length > 0 ||
+    filterQuery.creator.length > 0 ||
+    filterQuery.language.length > 0;
+
+
+  const filterMultiTagsPayload = () => {
```

Comment:
This name can be made better as it is not only about the tags.

Reply:
okay.